x-pack/plugins/observability/public/pages/slos/components/card_view/slos_card_view.tsx

```diff
@@ -68,27 +68,29 @@ export function SloListCardView({
   }
 
   return (
-    <EuiFlexGrid columns={columns}>
-      {sloList.map((slo) => (
-        <EuiFlexItem key={`${slo.id}-${slo.instanceId ?? 'ALL_VALUE'}`}>
-          <SloCardItem
-            slo={slo}
-            loading={loading}
-            error={error}
-            activeAlerts={activeAlertsBySlo.get(slo)}
-            rules={rulesBySlo?.[slo.id]}
-            historicalSummary={
-              historicalSummaries.find(
-                (historicalSummary) =>
-                  historicalSummary.sloId === slo.id &&
-                  historicalSummary.instanceId === (slo.instanceId ?? ALL_VALUE)
-              )?.data
-            }
-            historicalSummaryLoading={historicalSummaryLoading}
-            cardsPerRow={Number(cardsPerRow)}
-          />
-        </EuiFlexItem>
-      ))}
+    <EuiFlexGrid columns={columns} gutterSize="m">
+      {sloList
+        .filter((slo) => slo.summary)
```

Comment:
@shahzad31 Ok I see you keep only the slos that have summary, which is ok. I am just trying to understand what is the purpose of doing so. Did you notice a bug without the filtering line you added? 

Can we  actually have slos that do not have summary? I had the impression that there will always be a summary, cause we are creating a temp summary document at the very beginning until the actual summary document is indexed, no?

Reply:
yeah but this is to prevent the same bug we are trying to fix, where we did optimistic update to make add an SLO entry in the UI.

Reply:
Ok I see!